contracts/test/TestStrategy.sol

```diff
@@ -12,6 +12,11 @@ import {BaseStrategy, StrategyParams, VaultAPI} from "../BaseStrategy.sol";
 contract TestStrategy is BaseStrategy {
     bool public doReentrancy;
 
+    // Some token that needs to be protected for some reason
+    // Initialize this to some fake address, because we're just using it
+    // to test `BaseStrategy.protectedTokens()`
+    address public constant protectedToken = address(0xbad);
```

Comment:
I'm not sure if I should be using a made up address literal here, or deploying an actual ERC20 token. 

I thought about the latter, but it requires some extra code on the python test side of things, so I wasn't sure if that made sense to do. Plus, in a real example, I assume this would actually be a literal (like [here](https://etherscan.io/address/0xCa8C5e51e235EF1018B2488e4e78e9205064D736#code)).

Or maybe there's a better way to do this?

Reply:
If it works, it works!